src/app/components/Uploader/index.stories.tsx

```diff
@@ -30,7 +30,7 @@ const Component = ({ service, blocks }: PropsWithChildren<ComponentProps>) => (
 );
 
 export default {
-  title: 'New Components/Uploader Embed',
+  title: 'Components/Uploader Embed',
```

Comment:
Is there a reason why we are calling this an 'embed'?

Reply:
The "block" name that we get from Ares is 'embed', so that's why I included the word embed here. But on reflection I think that was more of a working title and isn't consistent with the component. So I have renamed to be just 'Uploader'

Reply:
Or maybe uploader-cta? Do we plan to actually build the uploader itself later?